src/app/(pages)/home/_components/CallOutWrapper.tsx

```diff
@@ -6,14 +6,24 @@ import { AnnouncementDataType } from '@/lib/notion/announcements';
 
 export default function CallOutWrapper({ text }: { text: AnnouncementDataType }) {
   const [isVisible, setIsVisible] = useState(true);
+  const [isRendered, setIsRendered] = useState(true);
 
   const toggleVisibility = () => {
-    setIsVisible(!isVisible);
+    setIsVisible(false);
   };
 
-  return (
-    <div className={`${isVisible ? 'opacity-100' : 'opacity-0'} transition-opacity duration-300 ease-in-out`}>
+  const handleTransitionEnd = () => {
+    if (!isVisible) {
+      setIsRendered(false); // Unmount component when transition ends
+    }
+  };
+
+  return isRendered ? (
+    <div
+      className={`${isVisible ? 'opacity-100' : 'opacity-0'} transition-opacity duration-300 ease-in-out`}
+      onTransitionEnd={handleTransitionEnd}
+    >
       {<CallOut text={text} toggleVisibility={toggleVisibility} />}
     </div>
-  );
+  ) : null;
```

Comment:
This is a really smart way of doing it Omar. Kudos!

Also just a side note: I believe if you really wanted to do it the CSS way, you could've used `display: none` which is just the `hidden` class... @heyitsOK @Anniechika (or maybe you've tried it and it didn't work?)

![image](https://github.com/user-attachments/assets/65c811fc-b49b-4ab0-aabc-8ad41e1177e3)

Regardless, I prefer this way of doing it (the React way vs the CSS way), just because there's weirdness with the way that tailwind dynamically compresses its classes

goodjob!